server/lib/renderers/xlsx.js

```diff
@@ -93,6 +92,16 @@ function render(data) {
   return wb.writeToBuffer();
 }
 
+
+/*
+ find available data to write in the excel file
+ rows is the default key, if nothing found, this function get look at rowsDataKey
+ it return an empty array if no data found
+*/
+function find(data, _options) {
```

Comment:
This is an interesting function.  To avoid needless variables, let's just use [JavaScript's notation for default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters):
```js
function find(data, options = {}) {
```